src/App.css

```diff
@@ -1,38 +1,117 @@
-.App {
+body {
+  margin: 0%;
+}
+
+
+.main-div {
+  display: flex;
+  height: 100vh;
+  border: 0;
+  padding: 0;
+}
+
+
+.left-container {
+  background-color: white;
+  width: 100%;
+}
+
+
+.right-container {
+  background-color: #2FBCF7;
+  width: 100%;
+  position: relative;
+}
+
+.menu {
+  padding-top: 3.32%;
+  padding-left: 2.86%;
+}
+
+
+.smilies {
+  padding-top: 9.11%;
+}
+
+
+.i6-img {
+  bottom: 6.58%;
+}
+
+
+.app-store-button {
+  position: absolute;
+  bottom: 25%;
+  /* This to center an image with absolute position */
+  margin: auto;
+  left: 0;
+  right: 0;
   text-align: center;
+  z-index: 1;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+
+.social {
+  position: absolute;
+  text-align: center;
+  bottom: 20%;
+  /* This to center an image with absolute position */
+  margin: auto;
+  left: 0;
+  right: 0;
+  z-index: 1;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+
+.play {
+  position: absolute;
+  text-align: center;
+  bottom: 57%;
+  /* This to center an image with absolute position */
+  margin: auto;
+  left: 0;
+  right: 0;
+  z-index: 1;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+
+@font-face {
+  font-family: Open Sans Bold;
+  src: url('./assets/fonts/OpenSans-Bold.ttf');
 }
 
-.App-link {
-  color: #61dafb;
+
+@font-face {
+  font-family: Open Sans Regular;
+  src: url('./assets/fonts/OpenSans-Regular.ttf');
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+
+.app-title {
+  font-family: Open Sans Bold;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 27px;
+  letter-spacing: 3px;
+  color: black;
+}
+
+
+.app-subtitle {
+  font-family: Open Sans Bold;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 32px;
+}
+
+
+.description-text {
+  font-family: Open Sans Regular;
```

Comment:
is this working?

Reply:
Yes, I defined a font-family, and used it setting the downloaded font